src/app/modules/item/components/item-header/item-header.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="task-header-container">
+  <div class="task-header">
+    <div class="task-user-image">
+      <img src="assets/images/_messi.jpg" style="width: 4rem; height: 4rem;" />
```

Comment:
ah and don't put an image of messi by default ;-)

Reply:
fixed, I removed the image all together